modules/gitops-installing-argo-rollouts-cli-using-mac-os.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-installing-argo-rollouts-using-mac-os_{context}"]
+= Installing Argo Rollouts CLI using Mac OS
+
+For macOS, you can install the Argo Rollouts CLI by using the link:https://brew.sh[Homebrew] package manager.
+
+.Prerequisites
+
+* You must have Homebrew (`brew`) installed.
```

Comment:
Same comment to prereqs as with Linux module.

Reply:
I can confirm this is the only prerequisite to install the CLI.